src/targets/services/checkAndUpdate.js

```diff
@@ -0,0 +1,35 @@
+import CozyClient from 'cozy-client'
+import { schema } from '../../helpers/doctypes'
+import log from 'cozy-logger'
+import {
+  updateContact,
+  getContactsToUpdate,
+  getLastSuccessOfService
+} from '../../helpers/updateContact'
+import { omit } from 'lodash'
+
+export const service = async () => {
+  log('info', `Executing checkAndUpdate service`)
+  const client = CozyClient.fromEnv(process.env, { schema })
+
+  try {
+    const lastSuccess = await getLastSuccessOfService(client, 'checkAndUpdate')
+    const contactsToUpdate = await getContactsToUpdate(client, lastSuccess)
+    const updatedContactsToUpload = contactsToUpdate.map(
+      // omit is to prevent updateAll error : Bad special document member: _type
+      contact => updateContact(omit(contact, '_type'))
+    )
+    await client
+      .collection('io.cozy.contacts')
+      .updateAll(updatedContactsToUpload)
+    updatedContactsToUpload.length &&
+      log('info', `All contacts successfully updated`)
+  } catch (e) {
```

Comment:
I do not think we need this catch here. In case of an error, the service will exit 0. I think crashing loudly in case of unknown error is better than crashing silently.